Python/password_strength_checker.py

```
@@ -0,0 +1,232 @@
+def test_passwords():
+    """Test various password examples"""
+    test_cases = [
+        "password",           # Very weak
+        "123456",             # Very weak  
+        "Password1",          # Weak
+        "Password123",        # Fair
+        "SecurePass123!",     # Good
+        "MyV3ryS3cur3P@ssw0rd2024!"  # Strong
+    ]
+    
+    checker = PasswordStrengthChecker()
```

Ruff (0.14.1)

Local variable `checker` is assigned to but never used

Remove assignment to unused variable `checker`

(F841)